pytrip/raster.py

```diff
@@ -240,9 +244,7 @@ def _generate_random_error_machine(self, sigma):
             for j, _ in enumerate(outgrid[0]):  # run along y indices
                 # only append to result grid _l, if we have a reasonable number of particles in this spot.
                 if outgrid[i, j] > 2000:  # spot threshold is 2000 particles
```

Comment:
extract that 2000 threshold to variables with proper comment - then it won't be a magic number, but just a number :)